chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CQMResultDetails.java

```diff
@@ -18,13 +19,17 @@
 
 import gov.healthit.chpl.dto.CQMResultCriteriaDTO;
 import gov.healthit.chpl.dto.CQMResultDetailsDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 
 /**
  * The clinical quality measure to which a given listing has been certified.
  */
 @XmlType(namespace = "http://chpl.healthit.gov/listings")
 @XmlAccessorType(XmlAccessType.FIELD)
 @JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
```

Comment:
Does this work? I thought there was something odd about the domain objects that are used as XML elements that means we can't use some of the Lombok annotations. I don't recall exactly what the problem is though

Reply:
As far as I know the odd thing is that we can't use the @Data annotation to replace getters and setters. I believe this does work.